polynote-kernel/src/main/scala/polynote/kernel/remote/RemoteKernel.scala

```diff
@@ -88,9 +87,10 @@ class RemoteKernel[ServerAddress](
     case rep: RemoteRequestResponse      => withHandler(rep)(_.run(rep).ignore)
   }
 
-  def init(): RIO[BaseEnv with GlobalEnv with CellEnv, Unit] = for {                                                          // have to start pulling the updates before getting the current notebook
-    pullUpdates    <- updates.evalMap(transport.sendNotebookUpdate).interruptAndIgnoreWhen(closed).compile.drain.forkDaemon   // otherwise some may be missed
```

Comment:
Is this comment ```have to start pulling the updates before getting the current notebook otherwise some may be missed``` no longer valid? why not?

Reply:
We still start pulling before getting the notebook. The comment was just lost.